raphtory/src/core/utils/hashing.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(dead_code)]
+
```

Comment:
why? should we remove the dead code instead or does it seem useful in the future?

Reply:
Its not actually dead_code - its used outside the lib and yeah are just useful functions